src/pages/ProductListing.js

```diff
@@ -0,0 +1,192 @@
+                                        </li>
+                                        <li>
+                                            <input type="checkbox" onChange={handleFilterChange} name="variety-filter" value="Pinot Noir" id="Pinot Noir" />
+                                            <label htmlFor="Pinot Noir" className="text-lg">Pinot Noir</label>
+                                        </li>
+                                    </ul>
+                                </details>
+                                : <details open>
+                                    <summary className="text-xl">Variety</summary>
+                                    <ul className="styled-list">
+                                        <li>
+                                            <input type="checkbox" onChange={handleFilterChange} name="variety-filter" value="Chardonnay" id="Chardonnay" />
+                                            <label htmlFor="Chardonnay" className="text-lg">Chardonnay</label>
+                                        </li>
+                                        <li>
+                                            <input type="checkbox" onChange={handleFilterChange} name="variety-filter" value="Sauvignon Blanc" id="Sauvignon Blanc" />
+                                            <label htmlFor="Sauvignon Blanc" className="text-lg">Sauvignon Blanc</label>
+                                        </li>
+                                        <li>
+                                            <input type="checkbox" onChange={handleFilterChange} name="variety-filter" value="Moscato" id="Moscato" />
+                                            <label htmlFor="Moscato" className="text-lg">Moscato</label>
+                                        </li>
+                                        <li>
+                                            <input type="checkbox" onChange={handleFilterChange} name="variety-filter" value="Pinot Grigio" id="Pinot Grigio" />
+                                            <label htmlFor="Pinot Grigio" className="text-lg">Pinot Grigio</label>
+                                        </li>
+                                        <li>
+                                            <input type="checkbox" onChange={handleFilterChange} name="variety-filter" value="Riesling" id="Riesling" />
+                                            <label htmlFor="Riesling" className="text-lg">Riesling</label>
+                                        </li>
+                                        <li>
+                                            <input type="checkbox" onChange={handleFilterChange} name="variety-filter" value="Viognier" id="Viognier" />
+                                            <label htmlFor="Viognier" className="text-lg">Viognier</label>
+                                        </li>
+                                    </ul>
```

Comment:
I feel like there is code repetition here, you could have used a separate component for cleaner code.